src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -144,7 +146,7 @@ public abstract class WebRequestPSCmdlet : PSCmdlet
         /// </summary>
         [Parameter(Position = 0, Mandatory = true)]
         [ValidateNotNullOrEmpty]
-        public virtual Uri Uri { get; set; }
+        public virtual Uri? Uri { get; set; }
```

Comment:
Consider [DisallowNull]

Reply:
What's the difference between [ValidateNotNullOrEmpty] and [DisallowNull] ?

Reply:
First is PowerShell attribute for parameters, second is .Net attribute for nullability annotations.

Reply:
Done